### ISSUE TITLE
Data sassOption

### ISSUE THREAD

**Issue by kvinlambert** [giuseppeg/styled-jsx-plugin-sass#18]
Hi,

Is there a way to use the data option from node-sass which would allow to prepend every styled-jsx block to import variables for example.

Right now this option doesn't seem to work for me:

```
"styled-jsx/babel",
{
    plugins: [
        ["styled-jsx-plugin-sass", {
            sassOptions: {
            includePaths: [
                path.resolve('node_modules', 'foundation-sites', 'scss'),
                path.resolve('node_modules', 'foundation-sites', '_vendor', 'sassy-lists', 'stylesheets', 'functions')
            ],
            data: `
            @import "src/assets/sass/imports";
            `
            }
        }
        ]
    ]
}
```

**Comment by giuseppeg** [comment]
if you use a version of styled-jsx and styled-jsx-plugin-sass that support options then I don't see why it shouldn't work. Are you using styled-jsx standalone or via some framework like Next.js? Can you prepare a minimal test app that I can clone and check out?

**Comment by madebyaaron** [comment]
Hi, so I happen to be experiencing this same issue, but it's worth noting that setting css styles via the data options works fine. The issue only arises when trying to import sass variables, mixins, functions etc.

````
// importing variables, doesn't work

plugins: [
    ["styled-jsx-plugin-sass", {
        sassOptions: {
            data: "@import "src/assets/sass/variables"; // $red: #00ffff
        }
    }]
]


// adding styles, does work

plugins: [
    ["styled-jsx-plugin-sass", {
        sassOptions: {
            data: "body { background-color: #00FFFF; }" // arbitrary styling here
        }
    }]
]
````

I hope this helps!

**Comment by Gameghostify** [comment]
Any updates on this? Would be awesome if we could get this to work

**Comment by kvinlambert** [comment]
Hmm @Gameghostify did you manage to get it working? Because I just tried your PR and it still won't work for me 😞

**Comment by giuseppeg** [comment]
just released v0.3.0 which includes this fix

**Comment by kvinlambert** [comment]
My bad @Gameghostify, my webpack config was wrong! Thanks for the fix! 👍
